### PR TITLE
Feat/create a help section 292

### LINKED ISSUE #292: Create a Help section
End-user documentation

- [x] How to create a memo for a token transfer?
- [x] How to use the spend tool?
- [x] How to categorize past TXs?
- [x] How to view the data?

### LINKED ISSUE #325: Changes Icons on Sidebar
Image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #292 
- Resolve #325 

### Steps to test

1. Run Project
2. Enter to Help
3. Must see this:
imagen
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: webapp/src/routes/Help/index.js [hunk 1]
```diff
@@ -1,68 +1,213 @@
...
+import { useTranslation, Trans } from 'react-i18next'
 
 import styles from './styles'
 
 const useStyles = makeStyles(styles)
 
 const Help = () => {
   const classes = useStyles()
-  const { t } = useTranslation('helpRoute')
+  const [expanded, setExpanded] = React.useState(false)
```
Comment (AngeloCG97) [inline webapp/src/routes/Help/index.js:13]: Please import useState from react

File: webapp/src/routes/IncomeReport/index.js [hunk 2]
```diff
@@ -47,6 +47,8 @@ const IncomeReport = () => {
     }
   ] = useIncomeReportState()
 
+  // console.log({ delegatesList })
```
Comment (AngeloCG97) [inline webapp/src/routes/IncomeReport/index.js:50]: Please remove this console log

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Review by btcurenas — APPROVED** [review]
LGTM Thanks

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**